tests/unit/GetTestResultsUtil.unitTest.ts

```diff
@@ -15,4 +16,46 @@ describe("GetTestResult Util functions", () => {
       expect(result).toEqual([{param1: "thing"}]);
     });
   });
+
+  describe("filterTestResultsByTestVersion", () => {
+    context("when testVersion is CURRENT", () => {
+      it("should return all test-results with testVersion=current or which don't have testVersion attribute", () => {
+        const myObject: any[] = [
```

Comment:
Maybe this can only be declared once since it seems myObject is the same in the the block?

Reply:
done